index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+      <a id="profile-link" href="https://codepen.io/pen?template=MJjpwO"
+         >My profile
+      </a>
+      <a class="links" href="#projects">Projects</a>	
+      <a class="links" href="#more">More</a>
+    </nav>
+   
+     <!-- welcome section -->
+   
+     <section id="welcome-section">
+       <h1>Hey guys...</h1>
+       <p>Lover of HTML and CSS</p>
+     </section>
+   
+     <!-- projects -->
+   
+     <section id="projects">
+        <a href="https://codepen.io/ks-97/pen/OJgqRjm">
+        <div class="project-tile">
+            <img src="https://images.unsplash.com/photo-1533749047139-189de3cf06d3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80"
+            alt="clock">
+            <p>
+            Project on a digital clock.
+            </p>
+        </div>
+        </a>
+     </section>
+    
+     <!-- footer -->
+  <footer id="more">
+    <div>@2021 Stay covid free.</div>
```

Comment:
I think this should have been a p tag instead of a generic div.

Reply:
- [ ] Instead of this you can write "Kindly wrap the line of code in `p tags`. It is recommended to use `p tags` for a line of code instead of `div` following the best HTML practices."

Reply:
Kindly wrap the line of code in p tags. It is recommended to use p tags for a line of code instead of div following the best HTML practices.